code/modules/clothing/neck/bodycamera.dm

```diff
@@ -0,0 +1,154 @@
+/*
+ * Bodycamera stuff
+ * NOTE: Unlike regular cameras, bodycams can only stream to one camera network at a time. Unless you want to give it more networks,
+ * the code will only ever worry about bodcam.network[1]
+ */
+
+/obj/item/clothing/neck/bodycam
+	name = "body camera"
+	desc = "A wearable camera, capable of streaming a live feed."
+	icon_state = "sec_bodycam_off"
+	item_state = "sec_bodycam_off"
+	var/prefix = "sec"//used for sprites, miner etc
+	var/obj/machinery/camera/bodcam = null
+	var/setup = FALSE
+	var/preset = FALSE //if true, the camera is already configured and cannot be reset
+	actions_types = list(/datum/action/item_action/toggle_bodycam)
+	strip_delay = 1 SECONDS //takes one second to strip, so a downed officer can be un-cammed quickly
+	w_class = WEIGHT_CLASS_NORMAL
+	var/mob/listeningTo //This code is simular to the code for the RCL.
```

Comment:
grumble gurmble grumble backreferences

not much u can do tho

Reply:
?

Reply:
what does this meean